doc/advanced.rst

```diff
@@ -546,6 +547,19 @@ from the token stream (``$this->parser->getStream()``):
 Parsing expressions is done by calling the ``parseExpression()`` like we did for
 the ``set`` tag.
 
+When encountering a syntax error during parsing, throw an exception::
+
+    throw new SyntaxError('Some error message.', $stream->getCurrent()->getLine(), $stream->getSourceContext());
+
+that follows the following best practices:
```

Comment:
I don't think splitting a sentence between 2 paragraphs with a code snippet between them is good for accessibility. Would it be possible to refactor this doc to avoid splitting the sentence ?

Reply:
Done